benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/models/BenchFlowTestNumber.java

```diff
@@ -14,7 +14,7 @@
 
   @Id private String testIdentifier;
 
-  private Long counter = 0L;
+  private Long counter = 1L;
```

Comment:
How do you use this number to increment the test number if the user has already executed a test with the same name?